### PR TITLE
Add underline and strikethrough to TextBlock

### PR DESCRIPTION
**Pull request by Patbox**
Adds basic support for underline and strikethrough within TextBlock.
image

### CODE REVIEW

**Comment by aWeirdo** [comment]
Cool, but can we fix the naming to match CSS "line-through" ?  
**lineThrough** in js

File: gui/src/2D/controls/textBlock.ts [hunk 1]
```diff
@@ -195,6 +197,42 @@ export class TextBlock extends Control {
...
 
+    /**
+     * Gets or sets an boolean indicating that text must have underline
+     */
+    public get underline(): boolean {
+        return this._underline;
+    }
+
+    /**
+     * Gets or setsan boolean indicating that text must have underline
```
Comment (deltakosh) [inline gui/src/2D/controls/textBlock.ts:208]: tiny typo

**Comment by deltakosh** [comment]
Also a tiny linting issue ;)

**Comment by deltakosh** [comment]
I will fix the lint no worries

**Pull request merged**